Software/bsw/gen/CanIf_Cfg.h

```diff
@@ -401,7 +405,7 @@ dependency: CANIF_CANPDUID_READDATA_API must be enabled */
 for reading its notification status.
 True: Enabled False: Disabled
 dependency: CANIF_READRXPDU_NOTIFY_STATUS_API must be enabled. */
-//#if(CANIF_PUBLIC_READ_RX_PDU_NOTIFY_STATUS_API==STD_ON)
+//#if(CANIF_PUBLIC_READRXPDU_NOTIFY_STATUS_API==STD_ON)
```

Comment:
Same as the 1st upper comments, we agreed that Macro's names are in UpperCase separated by a "_" character. so please use names in configuration files as they are.
also if you think there is anything to change in cfg files please request them in the cfg branches to be approved and pushed for the whole teams

Reply:
Because this parameter CANIF_PUBLIC_READRXPDU_NOTIFY_STATUS_API is the one used in AUTOSAR SWS as mentioned in page 54 and many other pages. Both formulas are used which is wrong and definitely creates bugs , So I decided to use only one formula